frontend/lib/src/components/elements/ImageList/ImageList.tsx

```diff
@@ -62,8 +62,8 @@ export function ImageList({
 
   const isSvgAllowed = React.useCallback(() => {
     if (hostConfig.disableSvgImages) {
-      throw new HostConfigViolation(
-        "Usage of SVG images is disabled by the security policy of the host."
+      throw new PlatformSecurityViolation(
+        "Usage of SVG images was removed in line with the platform security policy."
```

Comment:
nit: We probably can remove this check if we merge https://github.com/streamlit/streamlit/pull/7183. But since we are not blocking all SVG, we should probably add the unsafe word here:

```suggestion
        "The usage of unsafe SVG images was removed in line with the platform security policy."
```

Reply:
Yep once we merge #7183 , I will remove this check. However, since it's here, I changed it :) I'll add a TODO and commit that to make sure we don't miss it.